recipe/recipe.yaml

```diff
@@ -65,7 +64,10 @@ outputs:
   - package:
       name: ray-core
     build:
-      script: build-core
+      script:
+        file: build-core
+        env:
+          RAY_VER: ${{ version }}
```

Comment:
`PKG_VERSION` should already have the value we need if you want to simplify this